src/main/java/com/box/sdk/BoxGroupMembership.java

```diff
@@ -158,20 +163,41 @@ public Info(String json) {
         /**
          * Gets the level of access the user has.
          * @return the level of access the user has.
+         * @deprecated use getMembershipRole instead.
          */
+        @Deprecated
         public Role getRole() {
             return this.role;
         }
 
         /**
          * Sets the level of access the user has.
          * @param role the new level of access to give the user.
+         * @deprecated use setMembershipRole instead.
          */
+        @Deprecated
         public void setRole(Role role) {
             this.role = role;
             this.addPendingChange("role", role.toJSONString());
         }
 
+        /**
+         * Gets the level of access the user has.
+         * @return the level of access the user has.
+         */
+        public MembershipRole getMembershipRole() {
+            return this.membershipRole;
+        }
+
+        /**
+         * Sets the level of access the user has.
+         * @param role the new level of access to give the user.
+         */
+        public void setMembershipRole(MembershipRole role) {
```

Comment:
```suggestion
        public void setRole(MembershipRole role) {
```
This is less of a change, but does it really make things that much easier for a client adopting the change if we preserve the name of the deprecated method?

Reply:
Because of the comment for the getRole() I think we should also preserve this name to be consistent with get method name.